components/shared/LeftSidebar/CommandPalette/CommandPalette.tsx

```diff
@@ -0,0 +1,130 @@
+"use client";
+import React, { useEffect, MouseEvent } from "react";
+import { Command } from "cmdk";
+import Image from "next/image";
+import { postFilters, testTags } from "~/constants";
+import { useRouter } from "next/navigation";
+
+interface CommandPaletteProps {
+  isOpen: boolean;
+  setIsOpen: (isOpen: boolean) => void;
+}
+
+const CommandPalette: React.FC<CommandPaletteProps> = ({
+  isOpen,
+  setIsOpen,
+}) => {
+  const router = useRouter();
+
+  useEffect(() => {
+    const down = (e: any) => {
+      if (e.key === "k" && (e.metaKey || e.ctrlKey)) {
+        e.preventDefault();
+        setIsOpen(!isOpen);
+      }
+      //check if the key pressed is the escape key
+      if (e.key === "Escape") {
+        setIsOpen(false);
+      }
+    };
+
+    document.addEventListener("keydown", down);
+    return () => document.removeEventListener("keydown", down);
+  }, [isOpen, setIsOpen]);
+
+  const handleOverlayClick = (event: MouseEvent<HTMLDivElement>) => {
+    event.stopPropagation();
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <div>
+      {isOpen && (
+        <div className="cmdk-overlay" onClick={handleOverlayClick}>
+          <div className="cmdk-modal" onClick={(e) => e.stopPropagation()}>
+            <Command className="bg-myBlack-900">
+              <div className="flex w-full justify-between gap-2 bg-myBlack-700 p-2 text-myWhite-300">
+                <div className="flex">
+                  <Image
+                    src="/search-icon.svg"
+                    alt="Search"
+                    width={12}
+                    height={12}
+                  ></Image>
+                  <Command.Input
+                    className="w-full bg-myBlack-700 p-2 text-myWhite-300"
```

Comment:
```suggestion
                    className="w-full bg-myBlack-700 outline-none p-2 text-myWhite-300"
```
Try something like this to get rid of that jarring outline that's overlapping your icon :)

![image](https://github.com/dalbanhi/git-note/assets/4108484/7db90302-33bb-4c3b-b839-9c0167257e52)